lib/srv/desktop/README.md

```diff
@@ -15,183 +15,55 @@ See [this appendix](#appendix-virtualbox-notes) if using VirtualBox.
 
 #### AD DS
 
-First, we need to install Active Directory Domain Services (AD DS). From Server
-Manager, in the top-right select `Manage > Add Roles and Features`.
-
-In the wizard, select:
-- Before you Begin: click `Next`
-- Installation Type: `Role-based or feature-based installation`
-- Server Selection: click `Next` (should be only one server - current one)
-- Server Roles: select `Active Directory Domain Services`, in the popup click `Add Features`
-- Features: click `Next`
-- AD DS: click `Next`
-- Confirmation: click `Install`, wait for completion
-- Results: click `Promote this server to a domain controller` blue link
-
-Another wizard (AD DS configuration) will open:
-- Deployment Configuration: select `Add a new forest`, type in the Root domain
-  name. Any DNS-like name will work, for example `example.com`, **write down
-  this name for later**. Click `Next`.
-- Domain Controller Options: type in a `Directory Services Restore Mode (DSRM)
-  password` and click `Next`.
-- DNS Options: click `Next`
-- Additional Options: wait for the NetBIOS name to be generated, **write it
-  down for later**, click `Next.`
-- Paths: click `Next`.
-- Review Options: click `Next`.
-- Prerequisites Check: wait for the check to pass and click `Install`.
-- Results: after ~10s, VM will restart
+First, we need to install Active Directory Domain Services (AD DS). Save the following file as `domain-controller.ps1`,
+replacing `$domain` with your desired domain name and `$adminPassword` with your Administrator account's password.
+
+```powershell
+$domain = 'teleport.dev'
+$adminPassword = 'teleport123!!'
+
+$netbiosDomain = ($domain -split '\.')[0].ToUpperInvariant()
```

Comment:
How about using `(Get-ADDomain).NetBIOSName` instead of doing string manipulation?

Reply:
I don't think that would work because the domain hasn't been created at this point.